app/src/main/java/com/duckduckgo/app/tabs/model/TabDataRepository.kt

```diff
@@ -216,9 +216,9 @@ class TabDataRepository @Inject constructor(
         }.join()
     }
 
-    override suspend fun deleteCurrentTabAndSelectSource() {
+    override suspend fun deleteCurrentTabAndSelectSource(tabId: String) {
```

Comment:
@marcosholgado I think we talked about changing this method name to remove the part of `CurrentTab` now that we are passing `tabId` as parameter. Are you ok with that?

Reply:
👍